yoga/Yoga.c

```diff
@@ -2197,6 +2197,7 @@ static void YGNodelayoutImpl(const YGNodeRef node,
     // If the main dimension size isn't known, it is computed based on
     // the line length, so there's no more space left to distribute.
 
+    bool sizeBasedOnContent = false;
```

Comment:
I think that for the min and max case we shouldn't take the content into account. That's why it isn't const/inlined.